content/articles/2019-08-lifecycle-0-1-0.Rmarkdown

```diff
@@ -0,0 +1,189 @@
+
+### Development stages
+
+```{r echo = FALSE}
+knitr::include_graphics("/images/lifecycle/lifecycle-experimental.svg")
+```
+
+This is a new feature that is in the very early stage of development. It is exported so users can start to use it and report feedback, but its interface and/or behaviour is likely to change in the future. It is generally best to avoid depending on experimental features.
+
+```{r echo = FALSE}
+knitr::include_graphics("/images/lifecycle/lifecycle-maturing.svg")
+```
+
+The interface and behaviour of a maturing feature has been roughed out, but finer details are likely to change. It still needs more feedback to find the optimal API.
+
+```{r echo = FALSE}
+knitr::include_graphics("/images/lifecycle/lifecycle-stable.svg")
+```
+
+A feature is considered stable when the author is happy with its interface and behaviour. Major changes are unlikely, and breaking changes will occur gradually, through a deprecation process.
+
+```{r echo = FALSE}
+knitr::include_graphics("/images/lifecycle/lifecycle-questioning.svg")
+```
+
+The author is no longer convinced that the feature is the optimal approach. However, there are no recommended alternatives yet.
+
+
+### Deprecation stages
+
+```{r echo = FALSE}
+knitr::include_graphics("/images/lifecycle/lifecycle-soft-deprecated.svg")
+```
+
+The author is no longer happy with a feature because they consider it sub-optimal compared to some other approach, or simply because they no longer have the time to maintain it. A soft-deprecated feature can still be used without hassle, but users should consider switching to an alternative approach.
+
+```{r echo = FALSE}
+knitr::include_graphics("/images/lifecycle/lifecycle-deprecated.svg")
+```
+
+The feature is likely to be discontinued in the next major release. Users should switch to an alternative approach as soon as possible.
+
+```{r echo = FALSE}
+knitr::include_graphics("/images/lifecycle/lifecycle-defunct.svg")
+```
+
+The feature can no longer be used. A defunct function is still exported, and a defunct argument is still part of the signature. This way an informative error can be thrown.
+
+You can insert the lifecycle badges anywhere in your documentation. First import the badges in your package:
+
+```{r, eval = FALSE}
+# At the time of writing, you need the development version of usethis
+remotes::install_github("r-lib/usethis")
+usethis::use_lifecycle()
+```
+
+Then use the Rd macro `\lifecycle{stage}`:
+
+```{r, eval = FALSE}
+#' \lifecycle{experimental}
+#' \lifecycle{soft-deprecated}
+```
+
+A good place to include the lifecycle badge of a function is at the top of the `@description` block. To document an argument, the badge can be inserted in the argument description.
```

Comment:
If you included any examples, using lifecycle for an argument would be a good candidate. Include at start of param docs or end? What's this going to look like?

Reply:
Now with examples.